src/server/services/buurt/datasets.ts

```diff
@@ -487,6 +505,40 @@ export function transformMeldingenBuurtResponse(
   return transformDsoApiListResponse(datasetId, config, { features });
 }
 
+export async function fetchLaadpalen(requestConfig: DataRequestConfig) {
+  const urls = [
+    requestConfig.url,
+    'https://map.data.amsterdam.nl/maps/oplaadpunten?SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature&TYPENAMES=ms:snel_beschikbaar&OUTPUTFORMAT=geojson&SRSNAME=urn:ogc:def:crs:EPSG::4326',
+  ];
+  const requests = urls?.map((url) => {
```

Comment:
```suggestion
  const requests = urls.map((url) => {
```

Optional chaining is hier niet vereist aangezien urls altijd bestaat.